.github/workflows/ci.yml

```diff
@@ -11,13 +11,7 @@ jobs:
     - uses: actions/checkout@v3
     - uses: actions/setup-node@v3
     - name: Install dependencies
-      run: >
-        yarn
-        && yarn add -D eslint-config-airbnb-base eslint-plugin-unicorn eslint-plugin-jest@^26.9.0 @typescript-eslint/eslint-plugin @typescript-eslint/parser typescript github:prettier/prettier#2.8.1
-        && (cd node_modules/prettier && git apply -v ../../.github/fix-prettier-14007-no-unstable-semicolon.patch)
-        && (cd node_modules/prettier && git apply -v ../../.github/fix-prettier-14008-no-space-after-unary-minus.patch)
-        && (cd node_modules/prettier && git apply -v ../../.github/fix-prettier-14034-no-property-name-lower-if-inside-variable-declaration.patch)
-        && (cd node_modules/prettier && git apply -v ../../.github/fix-prettier-value-with-comma.patch)
+      run: yarn && yarn add -D eslint-config-airbnb-base eslint-plugin-unicorn eslint-plugin-jest@^26.9.0 @typescript-eslint/eslint-plugin @typescript-eslint/parser typescript github:fomantic/prettier#ca4fc1130f
```

Comment:
I suggest to create a version tag instead of using the commit hash

Reply:
why?

Reply:
Because i want something being "finished" and a version tag declares that (yes, technically it's nothing else than a pointer to a specific commit).
I created a github release out of your tag

Reply:
why GH release vs. not pure git tag?

Reply:
For the same reason 😉

Reply:
And its made out of your tag anyway

Reply:
I have intentionally (originally) disabled releases as in my PoV it is only an additional release step not needed for anything. And the more many manual steps we need to do per release, the more mistakes we can introduce.

As long as it is an internal repo I do not see any advantage than having `main` branch HEAD as the latest internal stable version... But I do not mind.

Reply:
Lets do it as clean as possible and not getting sloppy about it (i understand you want to get this finished now). In case of a new version you would have to change the ci to the new commit anyway.
And i think the minimum of maintenance is doing a proper version history. (and the main head might gets corrupted by untested commits if not protected/checked itself (thats why i asked about the linter error in the last commit) Referring to a specific tag/release is just as using a package manager

And who know who might gets interested in this fork because you fixed sutff, others also stumpled about 😉